modernizer-maven-plugin/pom.xml

```diff
@@ -87,5 +87,10 @@
       <version>4.1.6.RELEASE</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.gaul</groupId>
```

Comment:
It looks like this pom was alphabetical. Can you move this below `assertj-core`?